### PR TITLE
AD-6 created view, added new fields to models

### PR DESCRIPTION
**Pull request by MrBorusLee**

### CODE REVIEW

File: adventures/articles/models.py [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 
 class Article(models.Model):
-    text = models.TextField()
\ No newline at end of file
+    """Article model"""
+
+    text = models.TextField()
```
Comment (MsBorusLee) [inline adventures/articles/models.py]: Why `CASCADE`?

File: adventures/articles/views.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
-from django.shortcuts import render
+from django.http import HttpResponse
 
-# Create your views here.
+
+def view(request):
+    return HttpResponse('OK')
```
Comment (MsBorusLee) [inline adventures/articles/views.py]: It's useless...

File: adventures/articles/models.py [hunk 3]
```diff
@@ -2,4 +2,7 @@
```
Comment (MsBorusLee) [inline adventures/articles/models.py]: I can comment on unchanged lines :D

File: adventures/articles/views.py (hunk 2 above)
Comment (MrBorusLee) [reply adventures/articles/views.py]: No.

**Pull request merged**